li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -40,6 +40,7 @@
   public static final String AUTO_COMMIT_INTERVAL_MS_CONFIG = ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG;
   public static final String AUTO_OFFSET_RESET_CONFIG = ConsumerConfig.AUTO_OFFSET_RESET_CONFIG;
   public static final String MAX_POLL_RECORDS_CONFIG = ConsumerConfig.MAX_POLL_RECORDS_CONFIG;
+  public static final String TOPIC_CREATION_POLL_INTERVAL_MS_CONFIG = "li.topic.creation.poll.interval.ms";
```

Comment:
kafka clients already have a max.metadata.age or something - wouldnt it be better to default to that?

Reply:
That makes sense. I'll use metadata.max.age.ms to control the poll interval as well.